doc/feature/genericEventLoading.md

```diff
@@ -0,0 +1,119 @@
+Generic event loading {#genericEventLoading}
+=====================
+
+For research and debugging purposes, some times we need to quickly visualize
+a set of events in 3D space. For example, in a neural circuit with synaptic
+connectivity, we are interested in visualizing the position of the post-synaptic
+cells; or for validation purposes, we want to check that the LFP computation is
+done correctly for a predefined set of events.
+
+Since the variety of use cases is so wide, each of them with its own
+particularities, we need to find a generic way of loading any predefined set of
+events.
+
+The solution chosen for this feature is the loading of events from file. This
+offers users flexibility to generate all types of events in a simple and generic
+way, so the tool doesn't need to be adapted to fit the particular needs of any
+situation.
```

Comment:
I guess this is a stupid question that's clear for those who have used Fivox before, but what do you mean by generic event? A x, y, z tuple plus a scalar value, or something else?